### PR TITLE
feat(axe-core-4.9.1): update to use axe-core 4.9.1

### PR DESCRIPTION
**Pull request by rohittarpara**
#### Details

This PR upgrades axe-core to 4.9.1 according to the instructions in CONTRIBUTING.md.

##### Motivation

feature work 🚀

#### Pull request checklist

- [x] PR title respects [Conventional Commits](https://www.conventionalcommits.org) (starts with `fix:`, `feat:`, etc, and is suitable for user-facing release notes)
- [x] PR contains no breaking changes, **OR** description of both PR **and final merge commit** starts with `BREAKING CHANGE:`
- [x] Verify PR title and final merge commit contain space after `:` for example `feat(feature): feature title` otherwise it will not be considered by semantic-release for release.
- [ ] (if applicable) Addresses issue: #0000
- [ ] Added relevant unit tests for your changes
- [x] Ran `yarn precheckin`
- [x] Verified code coverage for the changes made

### CODE REVIEW

**Review by madalynrose — CHANGES_REQUESTED** [review]
Per the instructions in `CONTRIBUTING.md`, I think we also need to update the `devDependencies` in `package.json` for `@axe-core/cli` and `@axe-core/puppeteer`. And then regenerate the resources. Additionally, `src/cli.test.ts` needs to be updated similar to how `src/index.test.ts` was updated.

**Comment by rohittarpara** [comment]
@madalynrose i updated version for @axe-core/cli and @axe-core/puppeteer. 
For the CONTRIBUTING.md i changed the version. Is there any other thing i need to update in CONTRIBUTING.md?

File: package.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
         "yargs": "^17.0.0"
     },
     "devDependencies": {
-        "@axe-core/cli": "^4.8.0",
-        "@axe-core/puppeteer": "^4.8.0",
+        "@axe-core/cli": "^4.9.1",
+        "@axe-core/puppeteer": "^4.9.1",
```
Comment (madalynrose) [inline package.json:26]: These are usually specified using the minor version number, with patch version at 0
```suggestion
        "@axe-core/cli": "^4.9.0",
        "@axe-core/puppeteer": "^4.9.0",
```

**Review by madalynrose — CHANGES_REQUESTED** [review]
one small adjustment but this looks good!

**Review by madalynrose — APPROVED** [review]

**Pull request merged**